docs/analytics: what to do when site using analytics.js

chalin opened the pull request.

chalin commented:
@nate-double-u @thisisobate @caniszczyk - more updates to the migration steps. I'll merge now, but comments are welcome.

nate-double-u reviewed (commented):
I'm a little late to this one 😅

Do we need the [.vscode/cspell.json](https://github.com/cncf/techdocs/pull/123/files#diff-d9f1a54e1f1a4c8324f4756f08a6af27671a3ae9312190d4a5fe981cc1722824) file?

Generally I like to keep IDE files out of repos unless there's a pretty good reason to include them, and I'm not sure that this file is adding to the docs/analytics change that was intended with this PR.

chalin commented:
> Do we need the [.vscode/cspell.json](https://github.com/cncf/techdocs/pull/123/files#diff-d9f1a54e1f1a4c8324f4756f08a6af27671a3ae9312190d4a5fe981cc1722824) file?

It would be nice to have a spelling checker run on most of the docs. This is a first step in that direction, assuming we'd like to adopt cSpell. I ran the spelling checker on the Analytics page. I added page-local dictionary entries to the top of the `docs/analytics.md` page, and I added the rest of the words to the repo-wide dictionary. Hopefully that sounds reasonable to you.

> Generally I like to keep IDE files out of repos

Me too.

> unless there's a pretty good reason to include them, and I'm not sure that this file is adding to the docs/analytics change that was intended with this PR.

Yes it was intentional, as I described above.
